### DISCUSSION TITLE
OK啊兄弟们全体目光向我看齐

### DISCUSSION THREAD

**Discussion by passbyDFX** [share121/inter-knot#5014] [灌水]
我是个帅比！！！

啊啊啊啊啊啊好~无~聊~
_**无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊无聊**_

我忏悔，我因为太帅被全校师生表白，我真是太出生了
啊啊啊啊啊我没有疯哈哈哈哈哈哈

朋友，
挚爱，
梦想，
当我继承了东方之名，我也同时明白了那个女人的一切
但在这个名字底下，却没有属于我的东西
我只是扮演一个死去多年的影子
可是，我为什么要扮演一个失败的影子?
没错，是我杀死了艾晞，我做到了，做到了那个女人没有做到的事!我已经超越了TA！
我没有疯，我战胜了世界
这才是我自己的力量
我自己的归宿
我自己的理想
这才是真正的东方晞
以东方之名，唱跳rap，篮球！

你相信IKUN么？
我很佩服第一个打篮球的人，说不定会被全网黑呢？
kunkun啊！
人总归是要去马尔代夫打篮球的啊！
我再说一遍
篮球，要开始加速了！！！

小时候看刻舟求剑，觉得这个人好傻。他难道不知道剑在河中央掉下了水，跑到河边已经找不到了吗？长大以后才明白，在岁月的长河里，多少人一次又一次跑到河边按照曾经刻下的某个节点，想找到已经失去的东西。失去的东西永远不会回来，因为你已经不是曾经的你了，故地重游，故人重逢，本身就是刻舟求剑

上联：仙舟守不住停云，梦中救不了流萤
下联：舰船守不住姬子，乐土救不了爱莉
中联：须弥守不住树王，枫丹救不了水神
横批：我什么都做不到，我什么都做不到

没了，不发癫了
啊啊啊啊啊啊我是个帅比！！！
呜呜呜呜

**Comment by passbyDFX** [comment]
今天的精神状态已久很神经呢
想在学校吊死自己（悲）
作业终于写完了

**Comment by Junjun280** [comment]
草
